FloatingTextField/FloatingTextField.swift

```diff
@@ -2,8 +2,8 @@
 import UIKit
 
 let CScreenWidth = UIScreen.main.bounds.size.width
-let AnimationDuration = 0.3
-let LeftPadding: CGFloat = 15.0
+let defaultAnimationDuration = 0.3
```

Comment:
I've fixed the capitalization, which was a bit awkward, and added more context using `default` as prefix.